src/OSPSuite.R/Services/DataImporterTask.cs

```diff
@@ -128,9 +144,31 @@ public ImporterConfiguration GetConfiguration(string fileName)
          }
       }
 
+      /// <summary>
+      /// Creates an emtpy configuration with the columns "Time" and "Measurement".
+      /// </summary>
+      /// <returns>A new configuration object</returns>
       public ImporterConfiguration CreateConfiguration()
       {
-         return new ImporterConfiguration();
+         ImporterConfiguration configuration = new ImporterConfiguration();
+
+         Column timeColumn = new Column()
+         {
+            Name = _columnInfos.First(ci => ci.IsBase()).DisplayName,
+            Dimension = _dimensionFactory.Dimension("Time"),
```

Comment:
Use constant define in core
And for unit use Dimension.defaultUnit. No magic string